### DISCUSSION TITLE
Shortcut not working in terminal on Ubuntu

### DISCUSSION THREAD

**Discussion by IlyaMar** [hluk/CopyQ#2154] [Q&A]
Hello,

I've set up a shortcut for CopyQ window and it works fine for all windows except linux terminal. I know that terminal has own shortcuts, and tried different key combinations not used by terminal, e g Alt + Q, Ctrl + /, even NumLock. In all cases shortcut worked as expected for other windows except terminal.

$ copyq --version
CopyQ Clipboard Manager 6.0.1
Qt: 5.15.2
KNotifications: 5.89.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: Ubuntu 22.04.1 LTS

This w/a did not help: https://copyq.readthedocs.io/en/latest/known-issues.html#known-issue-wayland

**Comment by hluk** [comment]
On Wayland, it is perhaps best to set the shortcuts manually in the desktop environment itself. The command to show the main application window is `copyq show`.
